.github/workflows/tests.yml

```diff
@@ -47,23 +42,29 @@ jobs:
       - name: Clone Blobber
         uses: actions/checkout@v1
 
-      - name: Build Blobber
-        run: |
-          docker network create --driver=bridge --subnet=198.18.0.0/15 --gateway=198.18.0.255 testnet0
-          ./docker.local/bin/build.base.sh
-          ./docker.local/bin/build.blobber.sh
-
-      - name: Run Blobber on gRPC  
+      - name: Install Postgres
         run: |
           ./docker.local/bin/blobber.init.setup.sh
           cd docker.local/blobber1
           ../bin/blobber.start_github.sh
           docker ps
-
       - name: Run Tests
         run: |
+          make local-build
+          sudo make local-run &
+          sleep 15
+          running=0
+          echo "checking blobber server status"
+          if [ $running -eq 0 ]; then
+            echo "checking blobber server status"
+            curl 127.0.0.1:5051 && $running=1 || sleep 2
+          fi
```

Comment:
I would prefer having a job to test the dockerised builds as this is what would actually get deployed on production.

Reply:
No. it is not deployed on production . it is only working on github action. docker image is used on production.

Reply:
And it is not a dockerize build. it is `go build` and `run local app` in background . so check port to verify http server if it is up. if not, tests code might not work.